public/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>webpack boilerplate</title>
+</head>
+<body>
+    <div id="todo-container">
+        <p id="todo-title">Today's Todo</p>
+        <input type="text" class="todo-input" placeholder="Add to your list...">
+        <ul id="todo-list">
+        </ul>
+        <button id="clear-button" type="button">Clear all Completed</button>
+    </div>
```

Comment:
- Instead of using div in this situation you can use section to make it more semantic.
for better understanding you can read more about semantic on this link. http://diveintohtml5.it/semantics.html